patient-dao/src/main/java/hu/kuncystem/patient/dao/user/JDBCUserGroupDao.java

```diff
@@ -69,6 +70,8 @@ public UserGroup mapRow(ResultSet rs, int row) throws SQLException {
     public List<User> getAllUserFromGroup(UserGroup group) {
         try {
             return jdbc.query(SQL_GET_USERS_FROM_GROUP, new JDBCUserDao.UserRowMapper(), group.getId());
+        } catch (EmptyResultDataAccessException e) {
+            return null;

```

Comment:
It's better to return an empty collection to avoid NPE (and too many null checks) on the client side